News-Android-App/src/main/res/values/strings.xml

```diff
@@ -181,12 +181,16 @@
 
     <string name="pref_general_search_in_title">Title</string>
     <string name="pref_general_search_in_body">Body</string>
+    <string name="pref_general_search_in_both">Both</string>
 
     <string-array name="pref_general_search_in" translatable="false">
+        <item>@string/pref_general_search_in_both</item>
         <item>@string/pref_general_search_in_title</item>
         <item>@string/pref_general_search_in_body</item>
     </string-array>
     <string-array name="pref_general_search_in_values" translatable="false">
+        <!-- The ordering here is weird as to not break users current setting -->
+        <item>2</item>
         <item>0</item>
         <item>1</item>
     </string-array>
```

Comment:
Unsure if I should do it as this, having the default (`both`) should be at the top of the list. This results in `pref_general_search_in_values` having the items in a weird order to not break users current settings.

I could change it so both is 0, title is 1 and body is 2, but this will shift users settings, title -> both, body -> title but the `pref_general_search_in_values` would be in order.

Reply:
I think it should be fine like this, yes 👍 Your comment in the code explains it nicely why it is in that order.